src/test/datascience/notebook/nonPythonKernels.vscode.test.ts

```diff
@@ -151,6 +173,21 @@ suite('DataScience - VSCode Notebook - Kernels (non-python-kernel) (slow)', () =
 
         assertHasTextOutputInVSCode(cell, '123456', 0, false);
     });
+    test('Can run a CSharp notebook', async function () {
+        // Disabled, as .NET interactive kernel does not run on CI.
+        // https://github.com/microsoft/vscode-jupyter/issues/4061#issuecomment-740188708
+        return this.skip();
+        this.timeout(30_000); // Can be slow to start Julia kernel on CI.
```

Comment:
Is the timeout necessary here? If it is,
```suggestion
        this.timeout(30_000); // Can be slow to start csharp kernel on CI.
```